pkg/omf/functions/bundle/omf.bundle.install.fish

```diff
@@ -17,9 +17,18 @@ function omf.bundle.install
       test -n "$name_or_url"; or continue
 
       set name (omf.packages.name $name_or_url)
-
+      
       if not contains $name $packages
         omf.packages.install $name_or_url;
+        
+          # require if not a theme
+          and begin 
+            test $type = package; and begin
+              require $name
+                or echo "Failed to require $name"
+            end
+            or echo "Skipped require $name"
+          end
```

Comment:
As no messages need to be displayed, I guess those echo lines could be omitted. What do you think?

```suggestion
          and begin
            test $type = package;
              and require $name;
              or true
          end
```

It would be useful informing of failures while requiring packages, but that's prolly a subject for another PR.

Reply:
Yes, I agree the second suggest code block is better. But I think we should inform if any failure while requiring occurred. Maybe a if block would be more readable?

Reply:
True. We're not just solving oh-my-fish/plugin-weather#38. Agreed 👍